src/openrct2/ride/shops/meta/CashMachine.h

```diff
@@ -46,6 +46,7 @@ constexpr const RideTypeDescriptor CashMachineRTD =
     SET_FIELD(ColourPresets, DEFAULT_FLAT_RIDE_COLOUR_PRESET),
     SET_FIELD(ColourPreview, { 0, 0 }),
     SET_FIELD(ColourKey, RideColourKey::CashMachine),
+    SET_FIELD(MusicUpdateFunction, DefaultMusicUpdate),
```

Comment:
Can't these be dropped? Same goes for the 6 other ones.

Reply:
You cant, not with the SET_FIELD method.

Reply:
But `MusicUpdateFunction` is initialized with `DefaultMusicUpdate` by default, right?